docs/Command_Reference.md

```diff
@@ -79,7 +79,7 @@ Note: Environment variables will be automatically injected into containers creat
 * `hokusai promote` - Update the Kubernetes deployment on production to match the deployment running on staging.
 * `hokusai refresh` - Refresh the project's deployment(s).
 * `hokusai history` - Print the project's deployment(s) history.
-* `hokusai diff` - Print a git diff between the tags deployed on production vs staging
+* `hokusai diff` - Print a git diff between the tags deployed on production vs staging, you can pass `pr_only` to only see PR's merge commits
```

Comment:
Here, I would write it as the command-line option format: `--pr-only` as that's how users will invoke it.  (As you may have noticed, the Click library automatically reformats hypens to underscores, and calls the function wrapped by `@cli.command` with the resultant variable name, i.e. `pr_only`, as you have correctly done otherwise :)